### PR TITLE
Add dashboard for "Graph Metric"

### PR DESCRIPTION
**Pull request by knakazono**
This patch adds the dashboard for "Graph Metric", which is opened from
the alarm tab of monasca-ui horizon dashboard.

### CODE REVIEW

**Comment by knakazono** [comment]
@matrixik @kornicameister @timothyb89 @witekest @haruki-yamanashi

**Comment by matrixik** [comment]
Please provide screenshot how its look like. I think adding it here as a comment is enough.

**Comment by knakazono** [comment]
Thanks for your comment. Here is the screenshot of the dashboard.

graph_metric

This dashboard can show the graph of the metric which is related with the alarm.

**Comment by timothyb89** [comment]
This is really cool, I see some use for this outside of Horizon. What's the URL format look like? e.g. `http://my-grafana-url:3000/<stuff>`

**Comment by knakazono** [comment]
Hi,
You can open this dashboard with the following link.
```
http://my-grafana-url:3000/dashboard/script/drilldown.js
```
You need to specify the metric name with key "metric", and you can add some dimensions as additional parameters in URL like this.
```
http://my-grafana-url:3000/dashboard/script/drilldown.js?metric=sample&dim1=val1
```

**Comment by knakazono** [comment]
!push grafana

**Comment by knakazono** [comment]
As far as I checked, the build above failed here.
```
npm install -g yarn && yarn
```

Probably, the following error occurred.
```
/src/github.com/grafana/grafana # yarn
yarn install v1.0.0
info No lockfile found.
[1/5] Validating package.json...
error grafana@4.1.0-pre1: The engine "node" is incompatible with this module. Expected version "4.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

"yarn" was updated 6days ago, so maybe it relates with this issue.
Does anyone know this issue and how to solve this?

**Comment by knakazono** [comment]
yarn which is lower than 0.28 works with nodejs v6.x, so I posted a new pull request to use yarn v0.27.
https://github.com/monasca/monasca-docker/pull/207

Could you review the above patch first?

**Comment by knakazono** [comment]
@matrixik I added the explanation in README. could you review the latest one?

File: grafana/README.md [hunk 1]
```diff
@@ -37,6 +37,14 @@ properties:
 Note that in the [Kubernetes environment][6], the URL should instead be
 `http://monasca-api:8070/`.
 
+The dashboard for monasca-ui, which is opened by "Graph Metric" of alarm
```
Comment (digitalfishpond) [inline grafana/README.md]: ...which is opened by clicking on 'Graph Metric" in the 'Alarm' tab,...

File: grafana/README.md [hunk 2]
```diff
@@ -37,6 +37,14 @@ properties:
...
+tab, is available. You can use this dashboard outside of monasca-ui with
+the following link.
+`http://your-grafana-url:3000/dashboard/script/drilldown.js`
+Specify the metric name with key "metric" and dimensions as additional
```
Comment (digitalfishpond) [inline grafana/README.md]: ...name with the key "metric" and...

File: grafana/README.md [hunk 3]
```diff
@@ -37,6 +37,14 @@ properties:
...
+parameters if necessary like below.
```
Comment (digitalfishpond) [inline grafana/README.md]: ...necessary as below

File: grafana/drilldown.js [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+/*
+ * Complex scripted dashboard
+ * This script generates a dashboard object that Grafana can load. It also takes a number of user
+ * supplied URL parameters (in the ARGS variable)
+ *
+ * Return a dashboard object, or a function
+ *
+ * For async scripts, return a function, this function must take a single callback function as argument,
+ * call this callback function with the dashboard object (look at scripted_async.js for an example)
```
Comment (digitalfishpond) [inline grafana/drilldown.js]: ...dashboard object (see scripted_async for...

**Comment by knakazono** [comment]
Hi @digitalfishpond 
Thanks for your comments! I'm not familiar with English as you know, so it's very helpful for me :)
I updated the patch, so please review it.

**Pull request merged**